assets/javascript/render.js

```diff
@@ -0,0 +1,104 @@
+function getPokeValues(response) {
+  fetchAjax();
+  var res = {
+    attack: response.stats[4].base_stat,
+    hp: response.stats[5].base_stat,
+    num: response.id,
+    image: response.sprites.front_default,
+    name: response.name,
+    type: response.types[0].type.name
+  }
+  return res
+} //will decide what data gets saved
+
+
+function addPokeToVariables(response) {
+  var poke = getPokeValues(response) 
+  pokeName = poke.name;
+  pokeHealth = poke.hp;
+  catchHealth = poke.hp;
+  pokeImage = poke.image;
+
+// to add pokemon
+
+//   var ref = database.ref().child("Users").child(userId.uid)
+//   ref.push({ 
+//             name: pokeName,
+//             health: pokeHealth,
+//             image: pokeImage
+//           });
+// }
+}
+
+
+$('#pokemonCollection').on("click", "button", function() {
+  $('#user').empty();
+  $('#pouch').css("display", "none");
+
+//loads the pokemon from the pokemonCollection into the user side of battlemode
+//loads specific data into battlemode
+//important
+
+  referenceId = $(this).attr("data-id");
+  var ref = database.ref().child("Users").child(userId.uid).child(referenceId);
+  ref.on("value", function(snapshot) {
+    var poke = snapshot.val()
+    userHealth = poke.hp
+
+    var nameEntry = $('<h3>').text(poke.name)
+    var healthEntry = $('<h2>').text(userHealth)
+    var imageEntry = $("<img class='pokeBattle'>").attr("src", poke.image);
+
+    $('#user').append(imageEntry, healthEntry, nameEntry)
+
+  }, function (error) {
+     console.log("Error: " + error.code);
+  });
+
+//loads specific data into battlemode
+//important
+
+  $('#catch').empty();
+  var nameEntry = $('<h3>').text(pokeName)
+  var healthEntry = $('<h2>').text(catchHealth)
+  var imageEntry = $("<img class='pokeBattle'>").attr("src", pokeImage)
+  $("#catch").append(imageEntry, healthEntry, nameEntry)
+
+  battleMode();
+})
+
+function loadPokemon() {
```

Comment:
why do you hate my render function so much :'(

Reply:
I'd don't hate it. I actually think it's a pretty cool idea. I just don't think it's application fits the problem at hand, however. If you can make it so its applicable to two different types of rendering (battle mode & pouch display--which need to be different at least from a standpoint of changeable health; & potentially a "you just collected X pokémon with Y stats" modal), then I say go for it. If you do, you need to take into consideration UI perspectives and the different types of displays we'll need. For instance the pouch may need different data presentation that the battle mode, and the image in the battle mode may be a different size than the image in the pouch. And Jane needs to be able to figure out how to make those changes/

I think it's easier to just render it two different ways; which will provide time for other cool features, but if you want to focus in on the render function I get it. Sometimes these things are worth figuring out.

I'm just trying to make things go smooth.